.eslintrc

```diff
@@ -4,7 +4,8 @@
   },
   "globals": {
     "_": true,
-    "angular": true
+    "angular": true,
+    "document": true
```

Comment:
Ah yea good call. I had meant to remove this global, but that directive wasn't actually possible without using browser document (vs angular $document). I've been meaning to re-do my ng-if replacement anyway. I'll put in an issue to fix this.